base/lnls-single-uid.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+#
+# Configure container filesystem so that single UID restriction is satisfied,
+# making ignore_chown_errors=True storage configuration work properly.
+
+echo "APT::Sandbox::User \"root\";" > /etc/apt/apt.conf.d/90-disable-sandbox.conf
```

Comment:
Maybe this is not enough, since some files maybe owned by `adm` by default. With the current implementation, APT will still complain about `term.log` file:

```
W: chown to root:adm of file /var/log/apt/term.log failed - OpenLog (22: Invalid argument)
```

Reply:
That's a warning though, right?

Reply:
Yes, it is. I wonder if there is anything else we need to configure to properly have all its important features properly working. It seems that logging will be somewhat broken.

I haven't done any research on the topic yet.

Reply:
Well, there's nothing we can do regarding the log file warning, as this [attempt to change group ownership is hard-coded][1]. But it is indeed harmless.

I could not see other files with different ownership when sub{u,g}id are defined. It seems [other sandbox options][2] do not need to be changed as well.

[1]: https://salsa.debian.org/apt-team/apt/-/blob/8a8fdea4c71fbddfda9f7d6a1d9ec5da68870e12/apt-pkg/deb/dpkgpm.cc#L1022
[2]: https://salsa.debian.org/apt-team/apt/-/blob/8a8fdea4c71fbddfda9f7d6a1d9ec5da68870e12/doc/examples/configure-index#L669-L685